lib/rubocop/cop/rspec/factory_bot/factory_class_name.rb

```diff
@@ -15,7 +20,9 @@ module FactoryBot
         #   factory :foo, class: 'Foo' do
         #   end
         class FactoryClassName < Cop
-          MSG = "Pass '%<class_name>s' instead of %<class_name>s."
+          # rubocop:disable Metrics/LineLength
+          MSG = "Pass '%<class_name>s' string instead of `%<class_name>s` constant."
```

Comment:
Please wrap.

Reply:
Sorry, what should I wrap?

Reply:
The message. Please take a loot at [an example](https://github.com/rubocop-hq/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/describe_method.rb#L23).

Reply:
I got it, thank you.

Reply:
Fixed via a8167f3b3262943d890a016afbac959a4bbdce41